project_5.html

```diff
@@ -309,15 +303,17 @@ <h4 class="mb-3 fw-light">Datasheet and Pinout Diagram</h4>
 			</p>
 			<h4 class="mb-3 fw-light">Using the DFPLayer Mini with Arduino</h4>
 			<p class="mb-3">
-				You can download the DFPlayer Mini library <a class="link-primary"
-				href="https://drive.google.com/uc?export=download&id=1tc4KY-tViCPWNbc8Z_jIsfigmyeKCzg4" target="_blank"
-				rel="noopener noreferrer">here</a>. It can be used in your code with the following include:
+				You can download the DFPlayer Mini library v1.0.5 
+				<a class="link-primary"href="assets\libraries\DFRobotDFPlayerMini-1.0.5.zip">here</a>. 
+				We recommend this version as newer ones have produced errors in our testing.
+				It can be used in your code with the following include:
 			</p>
 			<p class="mb-3">
 				<code class="pj-code language-arduino"> #include &lt;DFRobotDFPlayerMini.h&gt;</code>
 			</p>
 			<p class="mb-3">
-				Below is sample code that demonstrates the module in serial mode. The program plays a audio file named “0001.mp3” for five seconds, then pauses it for five seconds, then resumes. Use this as a <strong>reference for the commands to send to the MP3 module</strong>.
+				Below is sample code that demonstrates the module in serial mode. The program plays a audio file named “0001.mp3” for five seconds, then pauses it for five seconds, then resumes.
+				Then, in the loop, it cycles through songs every 5 seconds. Use this as a <strong>reference for the commands to send to the MP3 module</strong>.
 			</p>
 			<p class="alert alert-warning mt-2" role="alert">
 				Do not blindly copy-paste the sample code to your project deliverables. Read it, understand it, then adapt it to your own code.
```

Comment:
![image](https://github.com/uci-ieee/ops-webpages-2023-2024/assets/75911565/1773007f-cf51-4e07-95e7-b3239454b9dc)

This is part is buggy on mobile